README.md

```diff
@@ -44,7 +44,7 @@ To run the python tests:
 
 To run the javascript tests:
 
-`$ cd testing` and `$ docker-compose up --build` and `$ docker exec -it javascripttests make linttestjs`
+`$ cd testing` and `$ docker-compose up --build` and `$ docker exec -it javascripttests npm install` and `$ docker exec -it javascripttests make linttestjs`
```

Comment:
This list is getting quite long now - so maybe we separate it into dot points?

```suggestion
* Build the container:`$ cd testing` and `$ docker-compose up --build`
* If this is the first time you're running the container, install the `npm` dependencies: `$ docker exec -it javascripttests npm install`
* Run the tests:`$ docker exec -it javascripttests make linttestjs`
```

Reply:
testjs-debug command added to Makefile.
Readme instructions to run JavaScript tests are separated into dot points.